skfda/exploratory/visualization/_ddplot.py

```diff
@@ -127,4 +121,35 @@ def plot(
             color=color_aux_line,
         )
 
-        return fig
+        return self.fig
+
+    def n_samples(self) -> int:
+        """Get the number of instances that will be used for interactivity."""
+        return self.fdata.n_samples
+
+    def set_figure_and_axes(
```

Comment:
This should not be public.

Reply:
i just realised again that this is called from multiple display, should i call it _set_figure_and_axes and import it? the other day it was giving me an error, with utils methods it works but as this one is a class method i dont know

Reply:
Why is this called from multiple display?

Reply:
because the baseplot is already created, we do a copy of it and then modify where it is displayed